### PR TITLE
fix: join country name and alpha3 code if specified

### LINKED ISSUE #38: country name is displayed with - at end
If not passing alpha3Code in countries array then - is displayed after country name.
There should be a condition check if no alpha3Code is set in array then even - should not be displayed
Sorry my bad English
[Please go through this page](https://github.com/angular-material-extensions/select-country/blob/513175a059b04ae8de52fb6d6f370b7936681742/projects/angular-material-extensions/select-country/src/lib/mat-select-country.component.html#L15)

### PR DESCRIPTION
**Pull request by KingDarBoja**
Fixes #38 by using a custom pipe to join elements if defined.

Also applies prettier to some files.

### CODE REVIEW

**Comment by AnthonyNahas** [comment]
I am preparing a new release so that will may not be compatible with the new version.

I need to push the release first to merge this one after that

**Comment by KingDarBoja** [comment]
No worries, I can fix the merge conflicts with latest changes from master so this can be shipped with next release.

**Comment by KingDarBoja** [comment]
This branch is up to date with latest changes from master 🚀  should be good to go.

File: projects/angular-material-extensions/select-country/src/lib/join.pipe.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { Pipe, PipeTransform } from '@angular/core';
```
Comment (AnthonyNahas) [inline projects/angular-material-extensions/select-country/src/lib/join.pipe.ts:1]: awesome

File: src/app/app.component.ts [hunk 2]
```diff
@@ -59,20 +60,29 @@ export class AppComponent implements OnInit {
...
+      country: [
+        {
           name: 'Deutschland',
           alpha2Code: 'DE',
           alpha3Code: 'DEU',
-          numericCode: '276'
-        }, disabled: false
-      }]
+          numericCode: '276',
+          disabled: false,
```
Comment (AnthonyNahas) [inline src/app/app.component.ts:70]: `disabled` is  not part of the country interface 

it's related to the form control! 
the first version was correct

**Review by AnthonyNahas — CHANGES_REQUESTED** [review]

File: src/app/app.component.ts (hunk 2 above)
Comment (KingDarBoja) [reply src/app/app.component.ts:70]: Oops, I copied the wrong part trying to not undo the prettier 😆

**Review by AnthonyNahas — APPROVED** [review]
thank you

**Pull request merged**